omero/developers/cli/extending.txt

```diff
@@ -0,0 +1,21 @@
+Extensions
+----------
+
+Plugins can be written and put in the ``lib/python/omero/plugins``
+directory. On execution, all plugins in that directory are registered
+with the |CLI|. Alternatively, the "--path" argument can be used to point to
+other plugin files or directories.
+
+Thread-safety
+^^^^^^^^^^^^^
+
+The ``omero.cli.CLI`` should be considered thread-*un*\ safe. A single
```

Comment:
I don't think the emphasis works here and not sure about the compound. How about:

should *not* be considered thread-safe

or

should be considered *not* thread-safe